install/installer/pkg/components/ws-proxy/objects.go

```diff
@@ -41,14 +39,7 @@ var Objects = common.CompositeRenderFunc(
 				ServicePort:   SSHServicePort,
 			},
 		}
-		return common.GenerateService(Component, ports, func(service *corev1.Service) {
-			// In the case of Workspace only setup, `ws-proxy` service is the entrypoint
-			// Hence we use LoadBalancer type for the service
```

Comment:
What is the effect of removing this when we do run in GCP?

Reply:
None. We patch [ws-proxy.](https://github.com/gitpod-io/ops/blob/main/deploy/workspace/gitpod-up.sh#L262-L289)